### PR TITLE
WIP: templates and tests to submit schemas

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: real-schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Real-life schemas and examples
+
+Please submit a PR with the schema (and a valid example, if you have it) in a separate folder here using one of the provided templates - you can use JSON, YAML of JavaScript.
```
Comment (ucarion) [inline real-schemas/README.md]: Nit: "or" not "of".

File: test-schemas/ajv.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const Ajv = require('ajv');
+ajv = new Ajv({
```
Comment (ucarion) [inline test-schemas/ajv.js]: Can we use `var` here? Otherwise we're setting a global?

Comment (ucarion) [reply test-schemas/ajv.js]: Or `const`, even, since we're using Node 10?

**Review by ucarion — APPROVED** [review]
LGTM! Just a small pair of nits.

File: test-schemas/ajv.js (hunk 2 above)
Comment (epoberezkin) [reply test-schemas/ajv.js]: Yes missed it

**Pull request merged**